source/content.ts

```diff
@@ -134,6 +134,7 @@ import './features/list-prs-for-file';
 import './features/pr-branch-auto-delete';
 import './features/show-whitespace';
 import './features/linkify-symbolic-links';
+import './features/remove-zero-packages';
```

Comment:
Let's name this `hide-zero-packages`, I'll try to rename the other features to always use `hide` instead of `remove`